Without freeing the raid_cmd_head list before creating the raid menu,
GUI menu of creating/deleting an array got mess up.

References: bsc#1165878
Upstream: submitted https://sourceforge.net/p/iprdd/mailman/message/36984523/

Signed-off-by: Wen Xiong <wenxiong@...>
---
 iprconfig.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/iprconfig.c b/iprconfig.c
index 5f31beb..f74a70c 100644
--- a/iprconfig.c
+++ b/iprconfig.c
@@ -2980,6 +2980,8 @@ int hot_spare_screen(i_container *i_con)
 **/
 int raid_screen(i_container *i_con)
 {
+	free_raid_cmds();
+
 	return display_features_menu(i_con, &n_raid_screen);
 }
 
-- 
1.6.0.2
